common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -90,10 +88,10 @@ object UrlHelpers {
     }
 
     val acquisitionData = Json.obj(
-      "source" -> AcquisitionSource.GuardianWeb.originalName,
+      "source" -> "GUARDIAN_WEB",
       "componentType" -> (position match {
-        case NewHeader | OldHeader | AmpHeader | SideMenu | SlimHeaderDropdown => ComponentType.AcquisitionsHeader.originalName
-        case Footer => ComponentType.AcquisitionsFooter.originalName
+        case NewHeader | OldHeader | AmpHeader | SideMenu | SlimHeaderDropdown => "ACQUISITIONS_HEADER"
+        case Footer => "ACQUISITIONS_FOOTER"
```

Comment:
and here we're referencing https://dashboard.ophan.co.uk/docs/thrift/componentevent.html#Enum_ComponentType